### PR TITLE
Reorganize travis setup to use centralized check script

### PR DESCRIPTION
**Pull request by dontcallmedom**
also use new infrastructure in Travis

### CODE REVIEW

**Comment by dontcallmedom** [comment]
Please ignore for now, I'm using this PR as a test ground

File: W3CTRMANIFEST [hunk 1]
```diff
@@ -0,0 +1,5 @@
+webrtc.html?shortName=webrtc respec
+images/icestates.svg
+images/ladder-2party-simple.svg
+images/peerstates.svg
+webrtc.css
```
Comment (alvestrand) [inline W3CTRMANIFEST]: No terminating linefeed?

File: W3CTRMANIFEST [hunk 2]
```diff
@@ -0,0 +1,5 @@
+webrtc.html?shortName=webrtc respec
```
Comment (alvestrand) [inline W3CTRMANIFEST]: Who reads this file? Is it possible to describe that either in a comment or in README?

**Comment by alvestrand** [comment]
I'd like to make sure we can figure out what's going on even if Dom's hit by a bus.
Otherwise, looks cleaner than before, which is a Good Thing.

**Comment by dontcallmedom** [comment]
yeah, part of making this set up ready is writing documentation for it, and training at least @vivienlacourba on how it works.

File: W3CTRMANIFEST (hunk 2 above)
Comment (dontcallmedom) [inline W3CTRMANIFEST]: The file is read by the CI script, but will also be read ultimately by the automatic publication system; I was hoping to document that in the comments, but it doesn't look like the format allows for it at the moment.

**Comment by dontcallmedom** [comment]
should be ready to merge now, cf https://lists.w3.org/Archives/Public/public-webrtc-editors/2015Nov/0006.html

File: .travis.yml [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
+      - cmake-data
 install:
- - git clone https://github.com/w3c/respec.git
- - git clone https://github.com/dontcallmedom/webidl-checker.git
- - git clone https://github.com/dontcallmedom/widlproc
-# - git clone https://github.com/halindrome/linkchecker.git
- - cd widlproc && make obj/widlproc && cd ..
- - pip install html5lib lxml html5validator
- - sudo apt-get install libwww-perl libcss-dom-perl
+ - git clone https://github.com/w3c/webrtc-respec-ci.git
```
Comment (martinthomson) [inline .travis.yml]: Have you considered adopting a `git submodules` approach here?  I note that you have a similar problem for the respec-ci repo as well.  Using the latest is fine, but is this going to be stable?

Also, @dontcallmedom, you disabled issues on the other repo.  I was going to report some things for which I don't have the time to generate a PR.

Comment (vivienlacourba) [inline .travis.yml]: @martinthomson I enabled Issues on https://github.com/w3c/webrtc-respec-ci

**Pull request merged**